setup.py

```diff
@@ -129,6 +134,11 @@ def build_extension(self, ext: CMakeExtension):
     "pennylane>=0.30",
 ]
 
+
+pennylane_plugins = ["lightning.qubit = pennylane_lightning:LightningQubit"] #TODO needed even with Kokkos backend
```

Comment:
I wonder if there is a better way here. I guess what we do is map the device name to the associated module and class name. Maybe this can be pulled programmatically? Not sure if that is feasible.

Reply:
I have updated `setup.py` in that direction.